docs/swagger.yaml

```diff
@@ -12,17 +12,17 @@ schemes:
   - https
 
 securityDefinitions:
-  api_key:
-    type: apiKey
-    description: API password
-    name: api_password
-    in: query
-  
-  # api_key:
+  #api_key:
```

Comment:
Stale comment.

Actually API key can both be in the query or as a header. Header is preferred.